src/nestor/handlers/contacts.py

```diff
@@ -203,12 +204,16 @@ def __show_birthday(self, *args) -> str:
         return Colorizer.success(str(record.birthday))
     
     @input_error()
-    def _get_birthdays(self) -> str:
+    def __get_upcoming_birthdays(self, *args) -> str:
         """
-        Returns all upcoming birthdays
+        Returns all upcoming birthdays within the given number of days
         """
-        birthdays: Dict[str, datetime] = self.book.get_upcoming_birthdays()
-
+        if not args[0].isdigit() or int(args[0]) < 0:
```

Comment:
Better just to do
```
        days = int(args[0])
        birthdays = self.book.get_upcoming_birthdays(days)
 ```
 and if `IndexError` raised `input_error` will catch it, we just need to update message in case if days not specified at all
 if `args[0]` is not a number string the `ValueError` will be raised and we just need to specify error messsage for 
 `input_error`

Reply:
fixed